src/main/webapp/index.html

```diff
@@ -5,40 +5,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  
     <title>GPostCard</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/env.css">
+    <link rel="stylesheet" href="style/style.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBFO8JyMu6f5bvQQG3c9KfEBayfbISs1gc"></script>
 
   </head>
   <body onload="login()" id="index_body">
+
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="main-nav">
+      <a class="navbar-brand" href="index.html"> <img src="image/logo.png" class="navbar-logo" alt=""></a>
+
+      <ul class="navbar-nav">
+        <li class="nav-item"><a class="nav-link" href="index.html">Home</a></li>
+      </ul>
+    </nav>
     <!--if the page browses slowly-->  
     <p id="loading">L O A D I N G</p > 
 
     <div id="login_page"></div>
     <div id="index_content">
-      <div id="about">
-        <text id="about_text">
-          Get started with the <text id="gpostcard">GPostCard!</text><br/>
-          <br/>Type in the text for your postcard.<br/> 
-          Tick the location checkbox if you want to share your travels!<br/>
-          We will generate a perfect postcard for you.</br>
-          You just need to SEND it! <br/>
-        </text>
+      <div class="inline" id="about">
+        <main class='container'>
```

Comment:
please use a consistent quote in tag.
If you use double quote in the html file, keep using this style instead of single quote.